hardhat.config.js

```diff
@@ -114,3 +121,7 @@ if (argv.coverage) {
   require('solidity-coverage');
   module.exports.networks.hardhat.initialBaseFeePerGas = 0;
 }
+
+function hasFoundry() {
```

Comment:
This will fail if Foundry is not installed. Why not just adding `true` as default in `argv`?

Reply:
Does it ? My understanding is it should return false. 
(its frangio's code)

Reply:
If I do 
```
require('child_process').spawnSync('eahzoth', { stdio: 'ignore' }).error === undefined
```
I get a clean `false`.

Reply:
You're right, I thought for a second it'll fail after returning false because Foundry was required for compiling, but it's not the case.

> (its frangio's code)

very on point! haha